### PR TITLE
Integration of SQ in GH action

### PR DESCRIPTION
**Pull request by amelie-renard-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,3 +1,22 @@
+= C++ example project scanned on SonarCloud using GitHub Actions
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
= C++ example project scanned on SonarQube using GitHub Actions
```

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]
How do you make sure this example works on SQ?
I have two ideas (compatible with each other):
- start from the SonarCloud example, and replace/remove SonarCloud references
- use peach as the sonarqube server, then in a separate change replace it with a place holder

**Comment by amelie-renard-sonarsource** [comment]
> How do you make sure this example works on SQ?
> I have two ideas (compatible with each other):
> 
> * start from the SonarCloud example, and replace/remove SonarCloud references
> * use peach as the sonarqube server, then in a separate change replace it with a place holder

I use ngrok to point to my localhost
(and of course I take what is done for SC as a starting point)

**Comment by arseniy-sonar** [comment]
> > How do you make sure this example works on SQ?
> > I have two ideas (compatible with each other):
> > 
> > * start from the SonarCloud example, and replace/remove SonarCloud references
> > * use peach as the sonarqube server, then in a separate change replace it with a place holder
> 
> I use ngrok to point to my localhost
> (and of course I take what is done for SC as a starting point)

I mean, to make the PR as the difference between SC (on `main`) and SQ versions.

File: README.adoc [hunk 2]
```diff
@@ -1,3 +1,22 @@
+= C++ example project scanned on SonarQube using GitHub Actions
```
Comment (arseniy-sonar) [inline README.adoc:1]: Please, mention in the readme that the `build.yml` is not complete and the user needs to set the correct sonarqube URL (maybe give an example of a correct URL)

**Pull request merged**